bundles/com.zeligsoft.domain.dds4ccm.ui.errorhandlers/schema/com.zeligsoft.domain.dds4ccm.ui.errorhandlers.undesirableExceptions.exsd

```diff
@@ -0,0 +1,123 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.zeligsoft.domain.dds4ccm.ui.errorhandlers" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="com.zeligsoft.domain.dds4ccm.ui.errorhandlers" id="undesirableExceptions" name="Undesirable Exceptions"/>
+      </appinfo>
+      <documentation>
+         [Enter description of this extension point.]
```

Comment:
A few documentation comments would be nice. At least explain the purpose of the extension point.